docs/submission/submitted-data.md

```diff
@@ -8,107 +8,90 @@ Program members can track the progress of their program’s clinical and molecul
 
-#### Files to QC
-
-_Coming Soon:_ Once an analysis workflow is completed (and the corresponding donor is clinically complete), the analysis files will be available to program members first to QCs. These files can be downloaded using a [file manifest and the score-client](/docs/data-access/data-download), and will be available for an embargo period before being released to the public via the File Repository.
-
-#### Donors with Released Files
+Program progress is summarized within the following dashboard "cards":
 
-_Coming Soon:_ Once the files have passed the embargo period, a data release will occur. Donors who have BOTH core clinical completeness and analysis files available will be released to the public via the File Repository.
+![Dashbard Overviwe and Statistics](/assets/submission/program_stats_section.png)
 
-#### All Files
+### Overview Statistics
+- The top card aggregates important program statistics: 
+	- The **number of Registered donors**, with a progress bar showing the goal of % Registered/Committed donors
+	- The **percentage of registered donors with complete clinical data**. Once the minimum amount of required clinical data has been submitted for a donor, they are considered [clinically complete](/docs/submission/clinical-data-validation-rules#clinical-data-completion).
+	- The **percentage of donors with submitted raw reads for a matched tumour/normal pair**. Once a donor has both a tumour and matched normal raw sequencing data pair [submitted through Song](/docs/submission/submitting-molecular-data), they will be counted towards this percentage.
 
-_Coming Soon:_ The count of all files for the program, i.e. all clinical and molecular analysis files that have either been released to the public or are still in QC for the embargo period. A program member, having DACO approval, can download these files by using the [file manifest with the score-client](/docs/data-access/data-download).
+	> **A donor must be clinically complete before any of their molecular analysis files are released to the program members.**
+	
+	> **A donor must also have data for at least one T/N pair before submitted before molecular processing will begin**
 
-## Donor Release Summary
+### Donor Release Summary
 
-The status bar in this card shows how many program donors have fully released files (dark blue bar), partially released files (light blue bar) and no released files (white bar) out of the total number of donors committed to ARGO by the program.  
-![Dashbard Donor Release Summary](/assets/submission/dashboard-donor-release-card.png)
+_Coming Soon:_ The status bar in this card shows how many program donors have fully released files (dark blue bar), partially released files (light blue bar) and no released files (white bar) out of the total number of donors committed to ARGO by the program.  
```

Comment:
hmmm we removed this out of the later dashboard designs. but it's there now right, just not working?

Reply:
yes, so i just want to leave it for now?

Reply:
sure should be fine